src/components/Dashboard/FilingHistoryList.vue

```diff
@@ -636,7 +646,13 @@ export default class FilingHistoryList extends Mixins(
     if (index >= 0) this.togglePanel(index, this.historyItems[index])
   }
 
-  private async correctThisFiling (item: HistoryItemIF): Promise<void> {
+  private startCorrectionDialog (item: HistoryItemIF): void {
+    this.fileCorrectionDialog = true
+    this.correctionFiling = item
+  }
+
+  private async correctThisFiling (correctionType: CorrectionTypes): Promise<void> {
```

Comment:
It looks like you don't do anything with correctionType. It should be saved in the draft correction filing.